src/connection_string.ts

```diff
@@ -526,20 +518,20 @@ function validateLoadBalancedOptions(
   hosts: HostAddress[] | string[],
   mongoOptions: MongoOptions,
   isSrv: boolean
-): MongoParseError | undefined {
+): void {
```

Comment:
Can we annotate this function as throwing / add a comment?  The error information used to be codified in the return type, but now it's lost.

Reply:
done!